desktop-app/app/containers/StatusBarContainer/index.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React, {useEffect} from 'react';
+import {ipcRenderer} from 'electron';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import StatusBar from '../../components/StatusBar';
+import {toggleStatusBarVisibility} from '../../actions/statusBar';
+import {STATUS_BAR_VISIBILITY_CHANGE} from '../../constants/pubsubEvents';
+
+const StatusBarContainer = props => {
+  useEffect(() => {
+    ipcRenderer.on(STATUS_BAR_VISIBILITY_CHANGE, _ => {
+      props.toggleStatusBarVisibility();
+    });
```

Comment:
@falecci Can you please render a function to remove this event listener on unmount, like in - https://github.com/manojVivek/responsively-app/blob/master/desktop-app/app/components/DevToolsResizer/index.js#L57

Reply:
Done :)!